### PR TITLE
docs(zh): update zh docs from en docs

### PR DESCRIPTION
**Pull request by KimYangOfCat**
based 7651d506e473886984eab625797477d63bc69981 commit, I updated zh docs from en docs

### CODE REVIEW

**Comment by KimYangOfCat** [comment]
@Jinjiang  hi, jinjiang, do you have any sugesttion?

File: docs/guide/mutations.md [hunk 1]
```diff
@@ -96,8 +96,7 @@ import { SOME_MUTATION } from './mutation-types'
 const store = createStore({
   state: { ... },
   mutations: {
-    // we can use the ES2015 computed property name feature
-    // to use a constant as the function name
+    // we can use the ES2015 computed property name feature to use a constant as the function name
```
Comment (Jinjiang) [inline docs/guide/mutations.md:99]: 不建议修改英文版任何内容

File: docs/zh/api/index.md [hunk 2]
```diff
@@ -174,58 +172,66 @@ sidebar: auto
...
 
   默认情况下，新的处理函数会被添加到其链的尾端，因此它会在其它之前已经被添加了的处理函数之后执行。这一行为可以通过向 `options` 添加 `prepend: true` 来覆写，即把处理函数添加到其链的最开始。
 
-  ``` js
+  ```js
   store.subscribe(handler, { prepend: true })
   ```
-  要停止订阅，调用此方法返回的函数即可停止订阅。
+ 
+ `subscribe` 方法将返回一个 `unsubscribe` 函数，当不再需要订阅时应该调用该函数。例如，你可能会订阅一个 Vuex 模块，当你取消注册该模块时取消订阅。或者你可能从一个Vue组件内部调用 `subscribe`，然后不久就会销毁该组件。在这些情况下，你应该记得手动取消订阅。
```
Comment (Jinjiang) [inline docs/zh/api/index.md:203]:
```suggestion
 `subscribe` 方法将返回一个 `unsubscribe` 函数，当不再需要订阅时应该调用该函数。例如，你可能会订阅一个 Vuex 模块，当你取消注册该模块时取消订阅。或者你可能从一个 Vue 组件内部调用 `subscribe`，然后不久就会销毁该组件。在这些情况下，你应该记得手动取消订阅。
```

File: docs/zh/guide/getters.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ const store = createStore({
 
 ## 通过属性访问
 
-Getter 会暴露为 `store.getters` 对象，你可以以属性的形式访问这些值：
+getter 会暴露为 `store.getters` 对象，你可以以属性的形式访问这些值：
```
Comment (Jinjiang) [inline docs/zh/guide/getters.md:43]: 应保持开头大写，这里不是特指 API 名称

File: docs/zh/guide/mutations.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Mutation
+# Mutations
```
Comment (Jinjiang) [inline docs/zh/guide/mutations.md:1]: 保持单数

File: docs/zh/guide/testing.md [hunk 5]
```diff
@@ -119,15 +119,15 @@ const testAction = (action, args, state, expectedMutations, done) => {
...
+    testAction(actions.getAllProducts, null, {}, [
       { type: 'REQUEST_PRODUCTS' },
       { type: 'RECEIVE_PRODUCTS', payload: { /* mocked response */ } }
     ], done)
   })
 })
 ```
 
-如果在测试环境下有可用的 spy (比如通过 [Sinon.JS](http://sinonjs.org/))，你可以使用它们替换辅助函数 `testAction`：
+如果在测试环境下有可用的 spies (比如通过 [Sinon.JS](http://sinonjs.org/))，你可以使用它们替换辅助函数 `testAction`：
```
Comment (Jinjiang) [inline docs/zh/guide/testing.md:130]: 保持单数

**Review by Jinjiang — CHANGES_REQUESTED** [review]

**Comment by KimYangOfCat** [comment]
@Jinjiang  hi, jinjiang, thanks for your suggesttions, them has be applied.

**Review by Jinjiang — APPROVED** [review]

**Pull request merged**